check-requirements.in

```diff
@@ -1,7 +1,16 @@
 bandit
+darglint
 flake8
+flake8-annotations-complexity
+flake8-bugbear
+flake8-broken-line
+flake8-commas
 flake8-comprehensions
+flake8-expression-complexity
 flake8-import-order
+flake8-quotes
```

Comment:
Use single quotes for strings, except when it would *contain* a single quote.

Reply:
I will be triggering this one a lot, but okay, that'll teach me.

Reply:
It's configurable to whine the other way around I think, it's main goal is consistency :)

Reply:
The problem is my own consistency :sweat_smile:

Reply:
I didn't want to hilight that, but then that plugin might someday do you some triggering O:)